packages/extension-polkagate/src/hooks/usePoolBalances.ts

```
@@ -0,0 +1,110 @@
+      const active = member.points.isZero()
+        ? BN_ZERO
+        : (new BN(String(member.points)).mul(new BN(String(stashIdAccount.stakingLedger.active)))).div(new BN(String(bondedPool.unwrap()?.points ?? BN_ONE)));
+      const rewards = myClaimable as Balance;
+      let unlockingValue = BN_ZERO;
+
+      member?.unbondingEras?.forEach((value: BN) => {
+        unlockingValue = unlockingValue.add(value);
+      });
+
+      genesisHash === chain?.genesisHash && setPooledBalance({ balance: active.add(rewards).add(unlockingValue), genesisHash });
+      setRefresh && setRefresh(false);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)